src/Tribe/Main.php

```diff
@@ -1953,10 +1953,6 @@ public function add_admin_assets() {
 				// date picker
 				Tribe__Events__Template_Factory::asset_package( 'datepicker' );
 
-				// jQuery Timepicker
-				wp_enqueue_script( 'tribe-jquery-timepicker' );
-				wp_enqueue_style( 'tribe-jquery-timepicker-css' );
```

Comment:
Shouldn't this still be enqueued somewhere?

Reply:
@borkweb yes, but it gets loaded in the "Asset package" for `events-admin.js` when it's called via `tribe_asset()`

See here: https://github.com/moderntribe/the-events-calendar/blob/7084c49219f875252cb5a219f07c88444d5445e6/src/Tribe/Asset/Admin.php#L17